modules/alert/README.md

```diff
@@ -59,21 +59,38 @@ No modules.
+| **denominator_filter** | A filter that identifies a time series that should be used as the denominator of a ratio that will be compared with the threshold. If a denominator_filter is specified, the time series specified by the filter field will be used as the numerator.The filter is similar to the one that is specified in the MetricService.ListTimeSeries request (that call is useful to verify the time series that will be retrieved / processed) and must specify the metric type and optionally may contain restrictions on resource type, resource labels, and metric labels. This field may not exceed 2048 Unicode characters in length. | string | n/a | no |
 | **denominator_aggregations** | Object containing configuration for the denominator aggregations  | object(any) | n/a | no |
 | **aggregations** | Object containing configuration for the trigger, that specifies the alignment of data points in individual time series selected by denominator_filter as well as how to combine the retrieved time series together (such as when aggregating multiple streams on each resource to a single stream for each resource or when aggregating streams across all members of a group of resources).When computing ratios, the aggregations and denominator_aggregations fields must use the same alignment period and produce time series that have the same periodicity and labels.This field is similar to the one in the MetricService.ListTimeSeries request. It is advisable to use the ListTimeSeries method when debugging this field.  | object(any) | n/a | **yes** |
 
+### **policies.conditions.condition_monitoring_query_language** supported attributes
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:--------:|
+| **query** | The query that outputs a boolean stream. | string | n/a | **yes** |
+| **duration** | The amount of time that a time series must violate the threshold to be considered failing. Currently, only values that are a multiple of a minute--e.g., 0, 60, 120, or 300 seconds--are supported. If an invalid value is given, an error will be returned. When choosing a duration, it is useful to keep in mind the frequency of the underlying time series data (which may also be affected by any alignments specified in the aggregations field); a good duration is long enough so that a single outlier does not generate spurious alerts, but short enough that unhealthy states are detected and alerted on quickly. | string | n/a | **yes**
```

Comment:
In the module you have a default value of "0s", so it's not required. Also, add the default value